src/main/java/hu/ak/generics/cafeloop/dao/SubscriptionDao.java

```diff
@@ -104,4 +105,4 @@ public void deleteById(int id) {
             throw new RuntimeException(e);
         }
     }
-}
\ No newline at end of file
+}
```

Comment:
Ez az apró változtatás több helyen is ott van az egész projektben, de ez csak whitespace, nem kell velük foglalkozni.
Valószínűleg az lehet, hogy itt szóközökből tabulátor lett, vagy fordítva. IntelliJ-jel és Eclipse-szel is dolgoztam, szóval lehet az egyik tabot, a másik szóközöket használ.